Apps/Examples/Examples/Models/Examples.swift

```diff
@@ -84,33 +84,48 @@ public struct Examples {
+                type: AddOneMarkerSymbolExample.self),
+        Example(title: "Add Circle Annotations",
+                description: "Show circle annotations on a map",
+                type: CircleAnnotationExample.self),
+        Example(title: "Add Cluster Symbol Annotations",
+                description: "Show fire hydrants in Washington DC area in a cluster.",
+                type: SymbolClusteringExample.self),
+        Example(title: "Add markers to a map",
+                description: "Add markers that use different icons.",
+                type: AddMarkersSymbolExample.self),
+        Example(title: "Add Point Annotations",
+                description: "Show point annotations on a map",
+                type: CustomPointAnnotationExample.self),
+        Example(title: "Add Polylines Annotations",
+                description: "Show polyline annotations on a map.",
+                type: LineAnnotationExample.self),
+        Example(title: "Animate Marker Position",
+                description: "Animate updates to a marker/annotation's position.",
+                type: AnimatedMarkerExample.self),
+        Example(title: "Change icon size",
+                description: "Change icon size with Symbol layer.",
+                type: IconSizeChangeExample.self),
+        Example(title: "Draw multiple geometries",
+                description: "Draw multiple shapes on a map.",
+                type: MultipleGeometriesExample.self),
         Example(title: "Use a map & annotations with SwiftUI",
                 description: "Use the UIViewRepresentable protocol to wrap a MapView in a SwiftUI view.",
                 type: SwiftUIExample.self),
-        Example(title: "Add multiple annotations to a map",
-                description: "Add default and custom annotations to a map.",
-                type: MultiplePointAnnotationsExample.self),
-        Example(title: "Add view annotations",
-                description: "Add view annotation on a map click.",
-                type: ViewAnnotationBasicExample.self),
-        Example(title: "Add view annotations connected to features",
-                description: "Add view annotation anchored to a symbol layer feature.",
-                type: ViewAnnotationMarkerExample.self)
+        Example(title: "View annotation with point annotation",
+                description: "Add view annotation to a point annotation",
+                type: ViewAnnotationWithPointAnnotationExample.self),
+        Example(title: "View annotations: advanced example",
+                description: "Add view annotations anchored to a symbol layer feature.",
+                type: ViewAnnotationMarkerExample.self),
+        Example(title: "View annotations: basic example",
+                description: "Add view annotation on a map with a click.",
+                type: ViewAnnotationBasicExample.self)
```

Comment:
Does the order make sense? Should the basic example be listed first?

Reply:
Reordered.